backend/api/tests/test_views.py

```diff
@@ -1,14 +1,17 @@
 import pytest
 import vcr
-import requests
+from django.test import Client
+
+client = Client()
 
 @vcr.use_cassette()
-def test_api_root():
-    response = requests.get('http://localhost:8000/api/v1/')
+def test_api_root(client):
+    response = client.get('/api/v1/')
     assert response.status_code == 200
-    assert 'businesses' in response.text
 
-@vcr.use_cassette()
-def test_businesses_endpoint():
-    response = requests.get('http://localhost:8000/api/v1/businesses/')
+
+@vcr.use_cassette(record_mode='new_episodes')
```

Comment:
Yeah your cassettes should always be in the version control.

I don't see the `X-App-token` anywhere in the cassette file, although it did try to re-generate the cassette for me so that might be a separate issue. Did you re-record and re-push the test after you made this comment?

Also, I've always found the `new_episodes` mode has always been a little frustrating, maybe slightly buggy even. You're better off recording in `once` mode and then manually deleting your cassettes when you need / want to re-record them.